### PR TITLE
SAA-296: Upgrade to Spring Boot 3

### PR DESCRIPTION
**Pull request by gerryallan-moj**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -17,19 +17,21 @@ configurations {
   testImplementation { exclude(group = "org.junit.vintage") }
 }
 
+repositories {
```
Comment (matt-crowson-moj) [inline build.gradle.kts:20]: Minor comment - probably worth adding a comment here to say why maven central is needed.

**Review by matt-crowson-moj — DISMISSED** [review]
LGTM with one minor comment,

File: build.gradle.kts (hunk 1 above)
Comment (gerryallan-moj) [reply build.gradle.kts:21]: comment added

**Review by matt-crowson-moj — APPROVED** [review]
LGTM

**Pull request merged**